build/config.gni

```diff
@@ -26,6 +26,10 @@ declare_args() {
   build_omaha = false
   tag_ap =""
   target_apk_base=""
+  brave_android_keystore_path = ""
```

Comment:
config args already exist for most of these
android_keystore_path, android_keystore_name and android_keystore_password

Reply:
If we specify `android_keystore_path` if will go through `package_apk` with errors. I specifically added `brave_` to not overlap with them